gpu-jobs.md

```diff
@@ -62,46 +74,60 @@ capabilities that are available to any CHTC user (as of 2/7/2020):
     <td>CentOS 7</td>
     <td>yes</td>
   </tr>
-  <tr>
-    <td>4</td>
-    <td>gzk-1 - gzk-4</td>
-    <td>8 </td>
-    <td>GeForce GTX 1080</td>
-    <td>SL 6</td>
-    <td>no</td>
-  </tr>
 </table>
 
-CHTC has plans to increase our GPU capacity through the [CHTC GPU
-Lab](/gpu-lab.shtml) project, funded by UW2020. This page will be
-updated as we acquire additional GPU capacity.
+### Job types, runtimes, and per-user limitations
+
+**Jobs running in the CHTC GPU Lab have different time limits than normal CHTC job
+submissions**. 
+
+See the table below for the default runtime limits for normally submitted jobs
+
+{:.gtable}
+  | Job type | Maximum runtime | Per-user limitation |
+  | --- |
+  | Short | 12 hrs | 2/3 of CTHC GPU Lab GPUs |
```

Comment:
I had a CTHC typo here and in the line below

Reply:
Should be CHTC, not CTHC

Reply:
I'll add these comments to #55 during my review since this pull request is already merged.